### ISSUE TITLE
Ender chest Deletes Items

### ISSUE THREAD

**Issue by johnwolf01** [scalda/FoolCraft#855]
## SUGGESTION/HELP
- [ ] I would like to propose an additional mod, config change, custom recipe, etc.
- [ ] I am requesting help with installing or running the pack

## Issue Description
When you are in a different dimension than your ender chest, placing items into the ender pouch to be transferred into your storage, the items get deleted. (5hr mining session worth of ore's and resources) Gone ! 

## Reproduce Steps
Note This is in single player 
Ender chest connected to main storage system via Item Conduit. located in End Dimension FTB Chunkloaded.
Mining in Overworld. Place resources in Ender Pouch (of same color than chest mentioned)
Items show that they are being Transferred into main storage (colossal Chest). 
When Checking the storage no items have been received or transferred to colossal chest.

Re-tested same method connected to vanilla chest and other storage blocks. Test Failed.

**Comment by johnwolf01** [comment]
@scalda 

And if I did not learn my lesson,

This also happens with the Wireless Crafting with Infinity card installed.
ALL items that are sent back across Dimensions into storage via these methods
Deletes the items and do not place into storage.

I mined about 2k worth of cobble, send it back into my storage via the wireless crafting grid .
I re-checked it and It showed that the items existed in my storage and was added, so I continued mining and sending, 3 times checked that it was there, and yes it was..

So after about 3 hrs of mining I had gathered enough resources and headed back to base.
Opened up the crafting grid to start building and crafting my next project..

*poof* - None of the cobble, redstone, diamonds, lapis, gold and iskallium that was mined was found.
all of it was voided even though I checked the qty's while in the overworld.

**Comment by scalda** [comment]
Have you got your ae system chunk loaded with the new witching gadgets mod?

**Comment by johnwolf01** [comment]
@scalda 
no but it is loaded with the FTB method. (the minimap that you higlight)

I dont know the mod that you are referring to I will look into it..

What bothers me thinking about it is... 
- 1) If the area is not loaded, items should not be "absorbed" into 
the system at all, and remain in the chest and not seemingly go into storage ... 

In the case of the wireless crafting device, it should not display or read the items inside the "AE system" because its in "unloaded chunks" - 
but the chunks are loaded (FTB chunk loading) and the storage are accessible in both cases. 

In other words the FTB Chunk loading is broken and should be disabled to avoid this issue and indirectly force players to use the other method.

Thank you Scalda !

**Comment by johnwolf01** [comment]
- Update
Using the Chunk loader from MFR - helps and solves this issue.
But it is Highly recommended that FTB chunk loading either are fixed or  Disabled.

Kind Regards

**Comment by scalda** [comment]
Ftb chunk loading is disabled has been since season 1. The pack now has
weirding gadgets as the chunloader which replaced chicken chunks

**Comment by johnwolf01** [comment]
Something strange.
one moment it works, then the next it does not.
not sure why.
Did several tests and it worked perfectly. sending stuff from the overworld , pop back to my base in the End, 

one thing that does come to mind is.
when logging out and you are not at your base. (the loaded chunks) 
you must return to your base first before sending anything to the remote dimension.

The chunks appear to be loaded since you can access the ME storage. but when you send items.
the items are actually not received. and hence deleted.
So it is very important for someone playing single player to visit his loaded chunks prior to sending anything into storage via remote.

I will add the weirding gadget re-test. 

Thank you Scalda

**Comment by johnwolf01** [comment]
The Weirding Gadget works good.
I have Both the gadget and the chunk loader of MFR just to be safe. :D 
Dont want to spend half a day mining only to lose it all .. ha ha ha .. it was so depressing.

Thank you Scalda
